### ISSUE TITLE
GPS hardware delays need better notice to users.

### ISSUE THREAD

**Issue by JavierRefuerzo** [UniversalDevicesInc/UD-Mobile-Android#46]
Sometimes GPS hardware has a delay of a minute or more before returning the users current location.  The current location is used to create new Geofences.  So the app needs to keep the screen on and ideally notify user that  the app is waiting on GPS location, maybe allow user to override to last known location or default location.

Another issue to consider here is that the GPS hardware is defective.  So allowing users to create Geofences may lead to other issues if the App/System never receives GPS coordinates.
